clients/lsp-rust.el

```diff
@@ -585,6 +585,13 @@ The command should include `--message=format=json` or similar option."
 (lsp-defun lsp-rust--analyzer-run-single ((&Command :arguments?))
   (lsp-rust-analyzer-run (lsp-seq-first arguments?)))
 
+;; `lsp-interface' is unsuitable for ShowReference destructuring, as that code
+;; action yields an array of three elements, not a map.
+(lsp-defun lsp-rust--analyzer-show-references
+  ((&Command :title :arguments? [_uri _filepos references]))
```

Comment:
This should have different indent or should be on the same line with the function name

Reply:
Then the line would be > 80 columns. Is that preferred?

Reply:
You can put multi-line for the parameters right?
That would make it easier to distinguished between params and body of a function

Reply:
The options are:
```elisp
(lsp-defun lsp-rust--analyzer-show-references ((&Command
                                                :title :arguments?
                                                [_uri _filepos references]))
```
or
```elisp
(lsp-defun lsp-rust--analyzer-show-references ((&Command :title :arguments?
                                                         [_uri _filepos
                                                               references]))
```
, both of which require two more lines and are IMHO uglier. What do you think?

Reply:
Sorry for late reply, both look fine to me.
The current solution has same indent between args and body and that make it quite hard to distinguish them.

Reply:
@yyoncho which do you prefer?

Reply:
For me all three options are acceptable... Just want to mention that we do not enforce 80 symbols or at least there is no agreement on that ATM.

Reply:
Thank you, I'll keep your latter point in mind. Not changing the call here is the best option then, because it means no new changes have to made and reviewed.